atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraCliParser.java

```diff
@@ -44,7 +60,11 @@ public static int parseSystemAuthReplicationFromCqlsh(String output) throws Ille
         throw new IllegalArgumentException("Cannot determine replication factor of system_auth keyspace");
     }
 
-    public static int parseNumberOfUpNodesFromNodetoolStatus(String output) {
+    private String getReplicationRegex() {
```

Comment:
Same with this. It's a helper method so it would be nice to have it at the bottom.